### PR TITLE
Create File to clean Ride-Share-Trip data

### PR DESCRIPTION
**Pull request by minhahoang**
Cleaning Rideshare trip data

### CODE REVIEW

File: Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[23]:
+
+
+import csv
+import pandas as pd
+import numpy as np
```
Comment (sarahdudek) [inline Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py:9]: I think you're not using csv and numpy so you can delete the imports here.

File: Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+reader.dtypes
+
+
+# In[18]:
+
+
+#Change name
+
+reader.columns=("trip_id",'ride_type_id','start_timestamp','end_timestamp','duration_second','miles','start_census_tract','end_census_tract','start_community_area','end_community_area','fare','tip','extra_charges','trip_total','shared_trip_authorized','start_centroid_latitude','start_centroid_longitude','start_centroid_location','end_centroid_latitude','end_centroid_longitude','end_centroid_location')
```
Comment (LuisFlosi) [inline Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py:59]: We want to make sure the start and end timestamps are in datetime format as well as rounded down to the hour (e.g. 08:01:01 becomes 08:00:00)

Comment (minhahoang) [reply Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py:59]: Actually, when I read data through python. The data look like not being seperated correctly. Still need to add something into the sep='' part. Can we somehow take a look at this together?

Comment (LuisFlosi) [reply Cleaning_Code/Trips data-Chicago-Rideshare-Analysis.py:59]: Let's look at it on Sunday then!

**Comment by sarahdudek** [comment]
Will close to avoid confusion

**Pull request closed without merging**